src/runtime/vendor/github.com/diskfs/go-diskfs/filesystem/squashfs/finalize.go

```diff
@@ -1471,7 +1471,7 @@ func updateInodesFromDirectories(files []*finalizeFileInfo) error {
 		case *basicDirectory:
 			dir.startBlock = d.directoryLocation.block
 			dir.offset = d.directoryLocation.offset
-			dir.fileSize = uint16(d.directoryLocation.size)
+			dir.fileSize = uint16(d.directoryLocation.size + 3)
```

Comment:
Hopefully nobody re-vendors this in the future. Any chance of making this change upstream?

Reply:
This is accepted by upstream. https://github.com/diskfs/go-diskfs/pull/286